Gruntfile.js

```diff
@@ -157,19 +231,87 @@ module.exports = function(grunt) {
             'ui-common': {
                 src: 'build/test-coverage/lcov/**/*.info',
             }
-        }
+        },
+        // file concatenation
+        concat: {
+            options: {
+                banner: cssFileHeader,
+            },
+            // css for the "page" template
+            pageCss: {
+                src: pageCssFiles,
+                dest: kbaseStaticDir + 'css/page_concat.css',
+                nonull: true,
+            },
```

Comment:
concatenate the files required for `page.html` / `notebook.html` into `page_concat.css`